src/indexing/EventIndexPeg.ts

```diff
@@ -27,11 +27,16 @@ import {SettingLevel} from "../settings/SettingLevel";
 
 const INDEX_VERSION = 1;
 
-class EventIndexPeg {
+export class EventIndexPeg {
+    public index: EventIndex;
+    public error: Error;
+
+    private _supportIsInstalled: boolean;
```

Comment:
do we need the underscore?

Reply:
Here we do, as there's a getter with the same name.